app/views/rails_performance/stylesheets/style.css

```diff
@@ -81,6 +81,9 @@
   border-bottom: 1px solid red;
 }
 
+.card-content {
+  overflow-x: scroll;
+}

```

Comment:
I thought this was included in my other pr that was merged. This change helped fix overflow issues on the tables in the UI.